lib/matplotlib/cbook.py

```diff
@@ -2244,6 +2244,21 @@ def _reshape_2D(X):
     return X
 
 
+def ensure_3d(arr):
```

Comment:
PEP8 doesn't really say what the boundaries of internal use are, but I guess here it means internal to matplotlib. Ok, I'll change this.